src/main/java/com/atlassian/db/replica/internal/ConsistencyGuardingChief.java

```diff
@@ -0,0 +1,29 @@
+package com.atlassian.db.replica.internal;
+
+import com.atlassian.db.replica.spi.Chief;
+import com.atlassian.db.replica.spi.ReplicaConsistency;
+
+import java.sql.Connection;
+import java.util.Collection;
+import java.util.function.Supplier;
+
+import static com.atlassian.db.replica.api.reason.Reason.REPLICA_INCONSISTENT;
+
+public class ConsistencyGuardingChief implements Chief {
+    private final ReplicaConsistency replicaConsistency;
+
+    public ConsistencyGuardingChief(ReplicaConsistency replicaConsistency) {
+        this.replicaConsistency = replicaConsistency;
+    }
+
+    @Override
+    public void overrideDecision(
```

Comment:
It looks strange. This gives a feeling that MultiReplicas aren't supported, and their behaviour needs to be overridden. 
I thought `Chief` should be used to override default decision.

I would rather add `withMultipleReplicas(Builder)` method to `DualConnection.Builder` to add support for them. WDYT?